src/index.js

```diff
@@ -48,7 +48,7 @@ module.exports = function (chai, utils) {
 
     utils.addMethod(chai.Assertion.prototype, "matchSnapshot", function (passedContext) {
         const actual = utils.flag(this, 'object');
-        const isForced = process.env.CHAI_SNAPSHOT_UPDATE || utils.flag(this, 'updateSnapshot');
+        const isForced = process.env.CHAI_SNAPSHOT_UPDATE || utils.flag(this, 'updateSnapshot') || process.argv.includes('--update');
```

Comment:
Instead of this can we make it `--snapshot-update`? I am thinking it should not conflict or confuse with existing or future possible mocha CLI flags

Reply:
Sure, good point.